Examples/RPC/rust/RUNME

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
```

Comment:
Nit: Can this filename be `runme.sh`? (Not applicable if `RUNME` is a Rust convention I do not know about.)

Reply:
I'm changing this to SETUP.sh (I don't know if this is a general pattern, but I used this ("SETUP.sh") for iOS samples)